packages/storage/amplify_storage_s3/example/windows/flutter/generated_plugins.cmake

```diff
@@ -4,6 +4,7 @@
 
 list(APPEND FLUTTER_PLUGIN_LIST
   amplify_db_common
+  connectivity_plus_windows
```

Comment:
connectivity_plus gets brought in by API plugin, s3 doesn't consume API though, why this gets here. Because aft bootstrap links everything?

Reply:
It's from using the `amplify_test` package